ethercat-sys/build.rs

```diff
@@ -4,87 +4,89 @@
 use std::{env, fmt::Write, fs, path::PathBuf};
 
 fn main() {
-    let path = env::var("ETHERCAT_PATH").expect(
-        "Please set the ETHERCAT_PATH env var to the location of \
+    if env::var(format!("CARGO_FEATURE_PREGENERATED_BINDINGS")).is_err() {
```

Comment:
Why `format!`?

Reply:
That's why I copied it from an other build file that used variables ;-)